### ISSUE TITLE
Sequence value when wrapper parsing

### ISSUE THREAD

**Issue by jakubfrydrych** [dailymotion/vast-client-js#357]
Hi, I want to ask if there is any way to get the sequence value when parsing each wrapper? In some cases, I would need to know this value, as one of the ad systems does not deliver a sequence for the end ad, so I would need to use the last or first sequence value.

Would it be possible to obtain information, for example, as part of an object for the VAST-resolving event VastParser?

Just for a quick problem preview
```
Wrapper sequence=2
  Wrapper sequence=null
    Wrapper sequence=null
      Ad sequence=null
``` 
I would be able to get sequence=2

```
Wrapper sequence=null
  Wrapper sequence=null
    Wrapper sequence=3
      Ad sequence=null
```

I would be able to get sequence=3

Thank you very much for your answer

**Comment by ZacharieTFR** [comment]
Hello @jakubfrydrych !
Sorry for the late reply,

According to IAB Specs https://iabtechlab.com/wp-content/uploads/2018/11/VAST4.1-final-Nov-8-2018.pdf#page=36 the `sequence` attribute is for `<Ad>` elements and is used for Ad Pods, which means that all ads with sequence attributes should be in the same VAST.
 
Unfortunatly you won't be able to get the value in the event VAST-resolving. 
But based on the `3.0-version` you could retreive the ad `sequence` value once parsing completed by checking the property "sequence" inside one the corresponding ad object in the array `ads` returned in the parsed VAST response.

I tested your uses case with the following VASTs : 
- Use case 1:  http://www.mocky.io/v2/5eb2d60632000005547b85ba
- Use case 2: http://www.mocky.io/v2/5eb2d3ea3200005c007b85ab

For both I was able to get the expected values. Let me know if you can reproduce and if it suits your needs.

**Comment by cizmo** [comment]
Hello @ZacharieTFR

Me and my colleague @jakubfrydrych working on this particular task together.
I don't think we gave you the best example on this. Our client needs to control ads order by inserting their own Wrapper Ad with sequence, because
they are not able to modify sequence of nested Ads directly.

In other words we need to get first sequence available when reading VAST.

For better clarification, we got a VAST with similar structure.

```
Wrapper
  Ad sequence=1
    Wrapper 
      Ad sequence=null
        Wrapper
          Ad sequence=3
            Wrapper
              Ad sequence=null
```

I slightly updated your use case 2 by adding `sequence` on first `<ad>`
https://run.mocky.io/v3/f21d1226-1ba9-4097-b75c-8f5238026532

Required result should be sequence 1.
Is it possible to somehow obtain this first sequence available? 

Thanks

**Comment by ZacharieTFR** [comment]
Hello @cizmo,
Indeed it make sense in the use case of an AdPod containing a wrapper. Thank you for clarifying !
Once https://github.com/dailymotion/vast-client-js/pull/363 is reviewed and merged, using `3.0-version` branch you will be able to retreive it in the parsed VAST response.

**Comment by ZacharieTFR** [comment]
Hello, 
Version 3.0 is officially [released](https://github.com/dailymotion/vast-client-js/releases/tag/3.0.0)! 🎉 
Also #363 has been merged and is available in the latest version.
